digiworld-epf/prompts.md

```
@@ -0,0 +1,155 @@
+First, I provided context to create the main prompt.
+
+
+As a developer with no experience in video game development, generate a prompt with indication for creating a video game.
+- The technology must be HTML, CSS, and JavaScript.
+- It must run from an index file.
+- The game must be a 2D video game with a character who: 
+1- Jumps between platforms
+2- Avoids obstacles
+3- Defeats enemies
+4- Advances through levels to a goal (only define 1 level)
+5- The back ground music mus be the one in this link :  https://fi.zophar.net/soundfiles/gameboy-advance-gsf/aladdin/30%20Unknown%20Song%201.mp3
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)